examples/fpn/README.md

```diff
@@ -0,0 +1,31 @@
+# Examples of Feature Pyramid Networks for Object Detection [1]
+
+## Performance
+MS COCO 2017 Val
+
+| Model | Original | ChainerCV |
+|:-:|:-:|:-:|:-:|
+| Faster-RCNN FPN ResNet50 | 36.7 % [2] | 37.1 % |
+| Faster-RCNN FPN ResNet50 | 39.4 % [2] | 39.5 % |
+
+Scores are mean of mean Average Precision (mmAP).
+
+## Demo
+Detect objects in an given image. This demo downloads MS COCO pretrained model automatically if a pretrained model path is not given.
+```
+$ python demo.py [--model faster_rcnn_fpn_resnet50|faster_rcnn_fpn_101] [--gpu <gpu>] [--pretrained-model <model_path>] <image>.jpg
+```
+
+## Evaluation
+The evaluation can be conducted using [`chainercv/examples/detection/eval_coco.py`](https://github.com/chainer/chainercv/blob/master/examples/detection).
+
+## Train
+You can train the model with the following code.
+Note that this code requires `chainermn` module.
+```
+$ mpi4exec -n <#gpu> python train_multi.py [--model faster_rcnn_fpn_resnet50|faster_rcnn_fpn_101] [--batchsize <batchsize>]
```

Comment:
mpiexec?

Reply:
README(s) of other models also use `mpi4exec`. So, I guess this is ok.

Reply:
I changed my mind since the official document of ChainerMN uses `mpiexec`. Should I update all README(s) in this PR?

Reply:
It is ok to change all READMEs in this PR.